packages/cardano-services/test/jest-setup/snapshots/stake_pool.sql

```diff
@@ -2655,6 +2667,15 @@ pool1m7y2gakwewqyaz05tvmqqwvl2h4jj2qxtknkzew0dhtuwuupreq	activating	191820000000
 \.
 
 
+--
+-- Data for Name: pool_delisted; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.pool_delisted (stake_pool_id) FROM stdin;
+pool1vj30jr7wn83dzn928qk6fx34h3d3f3cesr47j5ymeumf65wdw9x
```

Comment:
How was this record inserted before taking the snapshot?

If this was inserted with some _actions by hands_, next time we generate new fixtures, we will loose this.

Reply:
I manually insert it. Don't the fixtures run on top of what is already in the database?

Reply:
Removed in c91247ca2